.buildkite/scripts/packaging/package-step.sh

```diff
@@ -4,44 +4,44 @@ set -euo pipefail
 
 source .buildkite/env-scripts/util.sh
 
-changeset="^auditbeat/
+changeset="^${BEATS_PROJECT_NAME}/
 ^go.mod
 ^pytest.ini
 ^dev-tools/
 ^libbeat/
 ^testing/
-^\.buildkite/auditbeat/"
+^\.buildkite/${BEATS_PROJECT_NAME}/"
 
 if are_files_changed "$changeset"; then
   bk_pipeline=$(cat <<-YAML
     steps:
-      - label: ":ubuntu: Packaging Linux X86"
+      - label: ":ubuntu: ${BEATS_PROJECT_NAME}/Packaging Linux X86"
         key: "package-linux-x86"
         env:
           PLATFORMS: "+all linux/amd64 linux/arm64 windows/amd64 darwin/amd64 darwin/arm64"
-        command:
-          - ".buildkite/auditbeat/scripts/package.sh"
+          SNAPSHOT: true
+        command: ".buildkite/scripts/packaging/package.sh"
         notify:
           - github_commit_status:
-              context: "Auditbeat/Packaging: Linux X86"
+              context: "${BEATS_PROJECT_NAME}/Packaging: Linux X86"
         agents:
-          provider: "gcp"
+          provider: gcp
           image: "${IMAGE_UBUNTU_X86_64}"
 
-      - label: ":linux: Packaging Linux ARM"
+      - label: ":linux: ${BEATS_PROJECT_NAME}/Packaging Linux ARM"
         key: "package-linux-arm"
         env:
           PLATFORMS: "linux/arm64"
           PACKAGES: "docker"
-        command:
-          - ".buildkite/auditbeat/scripts/package.sh"
+          SNAPSHOT: true
+        command: ".buildkite/scripts/packaging/package.sh"
         notify:
           - github_commit_status:
-              context: "Auditbeat/Packaging: ARM"
+              context: "${BEATS_PROJECT_NAME}/Packaging: ARM"
         agents:
           provider: "aws"
-          imagePrefix: "${IMAGE_UBUNTU_ARM_64}"
-          instanceType: "t4g.large"
+          imagePrefix: "${AWS_IMAGE_UBUNTU_ARM_64}"
+          instanceType: "t4g.xlarge"
```

Comment:
What's the reason for these change? I'd like to see a comment with the rationale.

Reply:
Otherwise ARM packaging takes too long to build comparing to Ubuntu x86

Reply:
Thanks, could you add some details in the description about how much it takes in both cases?

I think those details could be useful in the future.